weasyprint/css/counters.py

```diff
@@ -295,6 +299,11 @@ def render_marker(self, counter_name, counter_value):
 
         value = self.render_value(counter_value, counter_name=counter_name)
         assert value is not None
+
+        if parent_direction == 'rtl':
+            suffix = symbol(counter['suffix'] or ('string', ' .'))
```

Comment:
Having ' .' instead of '. ' is an easy fix, but I’m not sure about how the specification handles this. We should read it, and maybe implement [`marker-side`](https://drafts.csswg.org/css-lists-3/#marker-side) to have a clean solution.

Reply:
I did not know marker-side specification was there. I am back with the specification but, thank you for pointing that out. I will try to implement it.